src/poprox_recommender/components/generators/context.py

```diff
@@ -9,7 +10,21 @@
 from poprox_recommender.lkpipeline import Component
 from poprox_recommender.topics import extract_general_topics
 
+from rouge_score import rouge_scorer
+from transformers import pipeline, AutoTokenizer, AutoModelForSequenceClassification
+
+# TODO: discuss if we need to unify all the language model usages in this project
+
 model = SentenceTransformer("all-MiniLM-L6-v2")
+# Define the nli model
+nli_model = pipeline("zero-shot-classification", model="facebook/bart-large-mnli")
+nli_labels=["entailment", "neutral", "contradiction"]
+
+# Load HateXplain tokenizer and model
+model_name = "Hate-speech-CNERG/dehatebert-mono-english"
```

Comment:
Do we need to follow-up with Karl about adding more models inside the docker container?